src/OrleansAzureUtils/AzureSilo.cs

```diff
@@ -150,29 +130,29 @@ public bool Start(string deploymentId, RoleInstance myRoleInstance, ClusterConfi
 
             myEntry = new SiloInstanceTableEntry
             {
-                DeploymentId = deploymentId,
-                Address = myEndpoint.IPEndpoint.Address.ToString(),
-                Port = myEndpoint.IPEndpoint.Port.ToString(CultureInfo.InvariantCulture),
+                DeploymentId = serviceRuntimeWrapper.DeploymentId,
+                Address = myEndpoint.Address.ToString(),
```

Comment:
Alternatively, I could rename `GetEndpoint()` to `GetIPEndpoint()` to be more explicit.